junit5/src/main/java/com/linecorp/armeria/testing/junit5/client/ThrowableAssert.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.testing.junit5.client;
+
+import static java.util.Objects.requireNonNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Assertion methods for Throwable.
+ */
+public final class ThrowableAssert {
+    private final Throwable cause;
+
+    ThrowableAssert(Throwable cause) {
+        this.cause = cause;
+    }
+
+    /**
+     * Verifies that the actual {@link Throwable} value is an instance of the given type.
+     * The {@code expectedType} cannot be null.
+     */
+    public ThrowableAssert isInstanceOf(Class<? extends Throwable> expectedType) {
```

Comment:
It would be useful if we can assert the cause of the current exception.
```java
public ThrowableAssert cause() {
    Throwable cause0 = cause.getCause();
    if (cause0 != null) {
        return new ThrowableAssert(cause0);
    } else {
        throw new AssertionError("Expecting actual throwable to have a cause but ..., ...);
    }
}

```